plugin/src/main/kotlin/com/vanniktech/maven/publish/MavenPublishBaseExtension.kt

```diff
@@ -18,13 +18,16 @@ import org.gradle.plugins.signing.Sign
 import org.gradle.plugins.signing.SigningPlugin
 import org.gradle.util.GradleVersion
 
-@Incubating
 abstract class MavenPublishBaseExtension(
```

Comment:
No need to have the extension itself still has incubating, some of it's methods already don't have it anymore.